### ISSUE TITLE
GetAuthIdCookie

### ISSUE THREAD

**Issue by eyal282** [alliedmodders/sourcemod#2007]
Why is there SetAuthIdCookie but not GetAuthIdCookie?

**Comment by asherkin** [comment]
What's your use case for getting a client preference for a client that isn't on the server?

**Comment by eyal282** [comment]
> What's your use case for getting a client preference for a client that isn't on the server?

Statistics, turning the default preference into the most used preference.

**Comment by Kenzzer** [comment]
I have to agree that it's strange `SetAuthIdCookie` but no `GetAuthIdCookie`. The existence of `SetAuthIdCookie` is imo a problem, because it requires some external source to provide the ids in order to be remotely useful, making its use case limited.

While I'd argue `GetAuthIdCookie` has more merit in existing, as plugins could be using cookies to establish leaderboards or something similar. And just for parity sake, I think `SetAuthIdCookie` needs its sibling `GetAuthIdCookie`.
